docs/attachments.md

```diff
@@ -27,11 +27,40 @@ The createJobToUploadBatchFile operation creates a job for batches that contain
 ```
 **Sample request**
 
-Following is a sample request that can be handled by the createJobToUploadBatchFile operation.
+Following is a sample request that can be handled by the createJobToUploadBatchFile operation and attach the file as form data.
```

Comment:
Following is a sample request that can be handled by the createJobToUploadBatchFile operation. It creates a job for batches that contain attachment records.

Reply:
Fixed in d9f4b3d6a3195a5dabee1246154595491561aa9b.